packages/features/voting-context.tsx

```diff
@@ -41,12 +66,24 @@ function VotingCtxProvider({ children }: ChildrenProps) {
     [partyList],
   );
 
+  useEffect(() => {
+    if (electionDatabaseId.length > 0) {
```

Comment:
Pode simplificar aqui mantendo a lógica!

```suggestion
    if (electionDatabaseId.length) {
```